<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sys_Sql_Help_Row_Wid</class>
 <widget class="QWidget" name="Sys_Sql_Help_Row_Wid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>880</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select the column(s) with Sql-Helper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/system_sys_resource.qrc">
    <normaloff>:/prom_icon</normaloff>:/prom_icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_ari">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Arithmetic and Ordering</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="Sys_Letter_Button" name="pushButton_plus">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="Sys_Letter_Button" name="pushButton_minus">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="Sys_Letter_Button" name="pushButton_div">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Sys_Letter_Button" name="pushButton_mult">
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="Sys_Letter_Button" name="pushButton_open_brack">
            <property name="text">
             <string>(</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Sys_Letter_Button" name="pushButton_close_brack">
            <property name="text">
             <string>)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="Sys_Letter_Button" name="pushButton_komma">
            <property name="text">
             <string>,</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_group">
         <property name="title">
          <string>Grouping</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="Sys_Letter_Button" name="pushButton_sum">
            <property name="toolTip">
             <string>Sum it up</string>
            </property>
            <property name="text">
             <string>SUM(</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Sys_Letter_Button" name="pushButton_count">
            <property name="toolTip">
             <string>Count the number of entrees</string>
            </property>
            <property name="text">
             <string>COUNT(</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="Sys_Letter_Button" name="pushButton_avg">
            <property name="toolTip">
             <string>Calculates the average</string>
            </property>
            <property name="text">
             <string>AVG(</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Sys_Letter_Button" name="pushButton_std">
            <property name="toolTip">
             <string>Calculates the standard deviation</string>
            </property>
            <property name="text">
             <string>STD(</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="Sys_Letter_Button" name="pushButton_max">
            <property name="toolTip">
             <string>Find the maximum value</string>
            </property>
            <property name="text">
             <string>MAX(</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="Sys_Letter_Button" name="pushButton_min">
            <property name="toolTip">
             <string>Find the maximum value</string>
            </property>
            <property name="text">
             <string>MIN(</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="Sys_Letter_Button" name="pushButton_astext">
            <property name="text">
             <string>ST_AsText(</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_row">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="title">
        <string>Column(s)</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_sql">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>209</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="send_button">
       <property name="text">
        <string>&amp;Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string>&amp;Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sys_Letter_Button</class>
   <extends>QPushButton</extends>
   <header location="global">sys_letter_button.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/system_sys_resource.qrc"/>
 </resources>
 <connections/>
</ui>
